image_scraper/utils.py

```diff
@@ -158,6 +158,23 @@ def matches_pattern(img_url):
 
         images = list(set(images))
         self.images = images
+        
+        css = tree.xpath("//link[@type='text/css']/@href")
+        css_images = []
+        for css_file in css:
+            if not re.match(r'^[a-zA-Z]+://', css_file):
+                css_file = self.url + css_file
+            image_list = re.findall('url\(([^)]+)\)', requests.get(css_file).content.decode('utf-8'))
+
+            for image in image_list:
+                if image.startswith('//'):
+                    image = 'https:' + image
```

Comment:
I think it should use the same as the URL put in to process ;) if not '//' is possible